src/liballoc/boxed.rs

```diff
@@ -774,6 +774,21 @@ impl<T: Copy> From<&[T]> for Box<[T]> {
     }
 }
 
+#[stable(feature = "box_from_array", since = "1.44.0")]
+impl<T, const N: usize> From<&[T; N]> for Box<[T]>
```

Comment:
This seems to be unsound for any T that isn't Copy.

Reply:
Should I do `impl From<[T; N> for Box<[T]>` instead?